clients/drcachesim/tests/burst_traceopts.cpp

```diff
@@ -37,6 +37,10 @@
  */
 
 #ifndef ASM_CODE_ONLY /* C code */
+
+// Enable asserts in release build testing too.
+#    undef NDEBUG
```

Comment:
Can we somehow do this in Cmake when building the test? This way we won't have to remember to do it for all new tests.

Reply:
Actually it turns out we removed CMake's default NDEBUG and then re-added it so yes.  Starting on this but hit some surprises.

Reply:
Done.